components/brave_wallet_ui/panel/container.tsx

```diff
@@ -626,74 +525,6 @@ function Container() {
     )
   }
 
-  if (selectedPanel === 'transactionDetails' && selectedTransactionId) {
-    return (
-      <PanelWrapper isLonger={false}>
-        <SelectContainer>
-          <TransactionDetailPanel
-            onBack={onGoBackToTransactions}
-            transactionId={selectedTransactionId}
-            visibleTokens={userVisibleTokensInfo}
-          />
-        </SelectContainer>
-      </PanelWrapper>
-    )
-  }
-
-  // Transactions
-  if (selectedPanel === 'activity') {
-    return (
-      <PanelWrapper isLonger={false}>
-        <StyledExtensionWrapper>
-          <Panel
-            navAction={navigateTo}
-            title={panelTitle}
-          >
-            <TransactionsPanel
-              selectedNetwork={selectedNetwork}
-              selectedAccount={selectedAccount?.accountId}
-            />
-          </Panel>
-        </StyledExtensionWrapper>
-      </PanelWrapper>
-    )
-  }
-
-  if (selectedPanel === 'assets') {
-    return (
-      <PanelWrapper isLonger={false}>
-        <StyledExtensionWrapper>
-          <Panel
-            navAction={navigateTo}
-            title={panelTitle}
-          >
-            <ScrollContainer>
-              <AssetsPanel
-                selectedAccount={selectedAccount}
-                onAddAsset={onAddAsset}
-              />
-            </ScrollContainer>
-          </Panel>
-        </StyledExtensionWrapper>
-      </PanelWrapper>
-    )
-  }
-
-  if (selectedPanel === 'sitePermissions') {
-    return (
-      <PanelWrapper isLonger={true}>
-        <LongWrapper>
-          <Panel
-            navAction={navigateTo}
-            title={panelTitle}
-          >
-            <SitePermissions />
-          </Panel>
-        </LongWrapper>
-      </PanelWrapper>
-    )
-  }
-
   if (selectedPanel === 'createAccount' && networkForCreateAccount) {
```

Comment:
networkForCreateAccount is always undefined

Reply:
Also it looks like we don't need `useHasAccount` hook any more

Reply:
Glad you noticed this! We actually need to handle this in the new `Panel V2`. Going to keep the `useHasAccount` hook and use it for this issue https://github.com/brave/brave-browser/issues/34300